Database.cpp

```diff
@@ -1,8 +1,44 @@
 #include "Database.hpp"
+#include <algorithm>
+
+//void searchByLastName();
+//void searchByPersonalID();
 
 void Database::printDatabase() const
 {
     for(const auto& person : data)
         std::cout << person;
     std::cout << "\n";
 }
+
+// void Database::sortBySalary();
+
+void Database::sortByLastName()
+{
+    std::sort(begin(data), end(data), [] (Person * left, Person * right)
```

Comment:
If you can, please stick * and & to type. 
Person * left -----> Person* left
int * a ------> int* a